workalendar/core.py

```diff
@@ -327,6 +327,19 @@ def get_whit_sunday(self, year):
     def get_corpus_christi(self, year):
         return self.get_easter_sunday(year) + timedelta(days=60)
 
+    def shift_christmas_boxing_days(self, year):
+        christmas = date(year, 12, 25)
+        boxing_day = date(year, 12, 26)
+        results = []
+        if christmas.weekday() in self.get_weekend_days():
+            shift = self.find_following_working_day(christmas)
+            results.append((shift, "Christmas Shift"))
+            results.append((shift + timedelta(days=1), "Boxing Day Shift"))
```

Comment:
I'd like to see the following label here: ``"{} Shift".format(self.boxing_day_label)``
In some countries, Boxing Day is named "St Stephen's" or "The Day After Christmas", for example